syncstorage/src/settings.rs

```diff
@@ -363,6 +379,15 @@ impl Settings {
         if let Some(max_age) = &self.cors_max_age {
             cors = cors.max_age(*max_age);
         }
+        // explicitly set the CORS allow origin, since Default does not
+        // appear to set the `allow-origins: *` header.
+        if let Some(origin) = &self.cors_allowed_origin {
+            if origin == "*" {
+                cors = cors.send_wildcard();
```

Comment:
Testing this locally I see this doesn't solve the issue. Looks like we want this:
```suggestion
                cors = cors.allow_any_origin();
```